File: 프로그래머스/lv0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.py
def solution(my_string):
    answer = []
    for i in my_string: # 문자열을 하나씩 나눠서 for문 돌리기
        if i.isdecimal(): # i가 0~9까지의 숫자인지 판별 => True라면
            answer.append(int(i)) # 해당하는 i를 int로 변환해서 answer에 추가
            answer.sort() # 오름차순 정렬
    return answer

# isdecimal() : 0~9의 숫자라면 True, 0~9의 숫자가 아니면 False
# isdigit(), isnumeric() : 숫자라면 True, 숫자가 아니면 False

File: 프로그래머스/lv0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.java
import java.util.Arrays;

class Solution {
    public int[] solution(String my_string) {
        
        String[] str = my_string.replaceAll("[^0-9]","").split(""); // 0-0가 아니면 공백으로 치환해서 split
        Arrays.sort(str); // 오름차순 정렬
        
        int[] answer = new int[str.length]; // answer 배열 선언 길이!!!
        for(int i=0;i<answer.length;i++){
            answer[i] = Integer.parseInt(str[i]); //int 형변환!!
        }
        return answer;
    }
}